kyaml/kio/byteio_writer.go

```diff
@@ -45,6 +45,9 @@ type ByteWriter struct {
 var _ Writer = ByteWriter{}
 
 func (w ByteWriter) Write(nodes []*yaml.RNode) error {
+	for _, node := range nodes {
```

Comment:
Revert these changes after writing the resources

Reply:
Done